### PR TITLE
Feature(content layout): implementation and minor tweak

### PR DESCRIPTION
**Pull request by rasmusThirslund**
# Content Layout

### CODE REVIEW

File: src/components/molecules/Navigation/Navigation.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ type NavigationProps = {
 const Navigation = memo(({ sidebar, sticky = false }: NavigationProps) => {
```
Comment (themitvp) [inline src/components/molecules/Navigation/Navigation.tsx:17]: Look what I found out, when you scroll in the table 😬 
image

Comment (rasmusThirslund) [reply src/components/molecules/Navigation/Navigation.tsx:17]: this is fixed now

File: src/features/settings/screens/UsersScreen/UsersScreen.tsx [hunk 2]
```diff
@@ -74,8 +74,7 @@ const UsersScreen = () => {
   };
 
   return (
-    <>
-      <ScreenTitleView title={t("settingsUsers.title")} />
+    <ContentLayout header={{ title: t("settingsUsers.title") }}>
```
Comment (themitvp) [inline src/features/settings/screens/UsersScreen/UsersScreen.tsx:77]: can you move `<UsersFilter />` up to `<ContentLayout filters={<UsersFilter />}` so the ContentLayout is used as intended? I forgot to add that 😊

Comment (rasmusThirslund) [reply src/features/settings/screens/UsersScreen/UsersScreen.tsx]: this is added now :)

File: src/features/settings/screens/UsersScreen/UsersScreen.module.scss [hunk 3]
```diff
@@ -1,7 +1,11 @@
 @import "styles/variables";
 
-.buttonContainer {
-  display: flex;
-  justify-content: flex-end;
-  margin: $spacing-4;
+.table {
```
Comment (themitvp) [inline src/features/settings/screens/UsersScreen/UsersScreen.module.scss:3]: this should be moved inside the `UsersTable` so it is closer to where it is actually being used

**Review by themitvp — APPROVED** [review]
super nice! 🎉

**Pull request merged**